relay-server/src/actors/project_upstream.rs

```diff
@@ -421,15 +440,27 @@ impl UpstreamProjectSourceService {
         });
     }
 
-    /// Handles the incoming external messages.
     fn handle_message(&mut self, message: UpstreamProjectSource) {
-        let UpstreamProjectSource(
-            FetchProjectState {
-                project_key,
-                no_cache,
-            },
-            sender,
-        ) = message;
+        match message {
+            UpstreamProjectSource::FetchProjectState(fetch_project_state, sender) => {
+                self.handle_project_source(fetch_project_state, sender)
+            }
+            UpstreamProjectSource::FetchGlobalConfig(global_config) => {
```

Comment:
nit: I would call this `UpdateGlobalConfig`. The handler of this message does not fetch anything.